src/piggui.rs

```diff
@@ -274,10 +289,13 @@ impl Application for Gpio {
                 pin_descriptions: None,     // Until listener is ready
                 pin_states: core::array::from_fn(|_| PinState::new()),
             },
-            Command::perform(Self::load(env::args().nth(1)), |result| match result {
-                Ok(Some((filename, config))) => Message::ConfigLoaded((filename, config)),
-                _ => Message::None,
-            }),
+            Command::batch(vec![Command::perform(
+                Self::load(env::args().nth(1)),
+                |result| match result {
+                    Ok(Some((filename, config))) => Message::ConfigLoaded((filename, config)),
+                    _ => Message::None,
+                },
```

Comment:
Just formatting, no?

We need to get to the bottom of which of us is not using the standard rustfmt - could be me, I'll try and check. Just check this out and run rustfmt and see if it changes I guess...

Reply:
yes, that's only formatting